web/admin/src/BigScreen.css

```diff
@@ -18,7 +18,7 @@
   opacity: 0.9;
 }
 .big-screen .box-content {
-  width: 75vh;
+  width: 75vw;
```

Comment:
Be sure to test out the Big Screen by itself out on different aspect ratios of windows (4:3) to (16:9), since `vh` was originally used to make sure different aspect ratios all worked